configs/sandpack-contents/component-theming/slider.js

```diff
@@ -0,0 +1,111 @@
+module.exports = {
+  App: `import {
+        Slider,
+        SliderTrack,
+        SliderFilledTrack,
+        SliderThumb,
+        Text,
+        Box,
+        Flex,
+      } from "@chakra-ui/react";
+        
+        export default function App() {
+          return (
+            <>
+            <Flex textAlign="center" gap={3} mt={3} direction="column">
```

Comment:
The examples are great, I just have one concern, the slider which is orange doesn't really go well with the blue, can you please change that?

Other than that, can you make the sliders have some padding so they don't take the whole screen.

And lastly if we can add the light/dark mode switch and I would add different styles when the dark mode is toggled.